File: app/src/main/java/org/flyontime/android/ui/adapter/CardType.java
package org.flyontime.android.ui.adapter;

/**
 * Created by jossi on 18.06.2017.
 */

public enum CardType {
    CHECKIN {
        @Override
        public String toString() {
            return "Check-in online";
        }
    },
    PREPARE {
        @Override
        public String toString() {
            return "Prepare for the trip";
        }
    },
    COMMUTE {
        @Override
        public String toString() {
            return "Travel to the airport";
        }
    },
    BAGGAGEDROP {
        @Override
        public String toString() {
            return "Baggage drop-off";
        }
    },
    SECURITY {
        @Override
        public String toString() {
            return "Pass security check";
        }
    },
    SHOP {
        @Override
        public String toString() {
            return "Shop and relax";
        }
    },
    BOARD {
        @Override
        public String toString() {
            return "Board on the plane";
        }
    }
}

File: app/src/main/java/org/flyontime/android/ui/adapter/CardType.kt
package org.flyontime.android.ui.adapter

/**
 * Created by jossi on 18.06.2017.
 */

enum class CardType {
    CHECKIN {
        override fun toString(): String {
            return "Check-in online"
        }
    },
    PREPARE {
        override fun toString(): String {
            return "Prepare for the trip"
        }
    },
    COMMUTE {
        override fun toString(): String {
            return "Travel to the airport"
        }
    },
    BAGGAGEDROP {
        override fun toString(): String {
            return "Baggage drop-off"
        }
    },
    SECURITY {
        override fun toString(): String {
            return "Pass security check"
        }
    },
    SHOP {
        override fun toString(): String {
            return "Shop and relax"
        }
    },
    BOARD {
        override fun toString(): String {
            return "Board on the plane"
        }
    }
}
